scheduler/cupsfilter.c

```diff
@@ -221,8 +227,10 @@ main(int  argc,				/* I - Number of command-line args */
 		i ++;
 		if (i < argc && !infile)
 		  infile = argv[i];
-		else
-		  usage(opt);
+		else{
```

Comment:
Coding style: CUPS puts open braces on their own line (not K&R), uses (traditional) 8 column tabs, and two spaces of indentation, e.g.:

```
else
{
  _cupsLanfPrintf(stderr, _("%s: Error - ..."), argv[0]);
  usage(NULL);
}
```